source/documentors/references/quick_reference_rst.rst

```diff
@@ -4,8 +4,13 @@ Quick Reference: Writing RST
 RST is very powerful and flexible.  Below we provide a quick guide for how you
 can use it to write Open edX documentation.
 
-For more details check out the `RST Primer`_
+.. seealso::
+
+   `A one-page reference document`_ (PDF)
+
+   For a deeper introduction, check out the `RST Primer`_
 
+.. _A one-page reference document: https://docs.openedx.org/en/latest/documentors/references/RST_quick_reference.pdf
```

Comment:
@feanil I don't know how to link this PDF. I added to the directory, but https://docsopenedxorg--286.org.readthedocs.build/en/286/documentors/references/RST_quick_reference.pdf doesn't show up. Is there another place to put a PDF / another way to display it?

Reply:
I think the [:download:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#referencing-downloadable-files) role is probably the way to go.

Reply:
Would I put the role here, or above? EG,

```
.. _A one-page reference document: :download:https://docs.openedx.org/en/latest/documentors/references/RST_quick_reference.pdf
```

or above, eg

```
:download:`A one-page reference document`_ (PDF)
```

I'll also change it back to a relative link, which wasn't working before, and see if the directive helps